### PR TITLE
fix for art-pi smart

### PR DESCRIPTION
**Pull request by xiao-mang**
## 拉取/合并请求描述：(PR description)

[

#### 为什么提交这份PR (why to submit this PR)

fix for art-pi smart

#### 你的解决方案是什么 (what is your solution)

modify some config

#### 请提供验证的bsp和config (provide the config and bsp) 

- BSP:rt-thread/bsp/nxp/imx/imx6ull-smart

- .config:please see the code

- action:

]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [ ] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [ ] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [ ] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [ ] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [ ] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [ ] 代码是高质量的 Code in this PR is of high quality
- [ ] 已经使用[formatting](https://github.com/mysterywolf/formatting) 等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

**Comment by geniusgogo** [comment]
## 当前支持情况

### 内核

- [ ] rt-thread
- [x] rt-smart

### 驱动

- [x] uart
- [x] SPI
- [x] GPIO
- [x] sdcard
- [x] RW007
- [x] enet
- [ ] usb
- [x] emmc

### 组件

- [x] fat文件系统
- [x] LWIP网络协议栈
- [x] wlan

### 应用

- [x] xmake 构建环境
- [x] busybox
- [x] smart-apps

### 待完善、修复

- [ ] ash xxxx
- [ ] 网络xxxxx
- [ ] emmc xxxxx

File: bsp/nxp/imx/imx6ull-smart/applications/mnt.c [hunk 1]
```diff
@@ -20,6 +20,7 @@ int mnt_init(void)
     rt_thread_mdelay(500);
 
     int part_id = 0;
+    #if 0
```
Comment (BernardXiong) [inline bsp/nxp/imx/imx6ull-smart/applications/mnt.c:23]: 不需要就移除掉

File: components/dfs/dfs_v2/src/dfs_file_mmap.c [hunk 2]
```diff
@@ -184,7 +184,7 @@ void page_read(struct rt_varea *varea, struct rt_aspace_io_msg *msg)
                varea->start, varea->size, varea->offset, varea->attr, varea->flag);
 
         ret = dfs_aspace_mmap_read(file, varea, msg);
-        if (ret > 0)
+        if (ret >= 0)
```
Comment (geniusgogo) [inline components/dfs/dfs_v2/src/dfs_file_mmap.c:187]: 这个修改和下面pcache那个修改单独提一个PR

**Review by geniusgogo — CHANGES_REQUESTED** [review]

File: bsp/nxp/imx/imx6ull-smart/applications/mnt.c (hunk 1 above)
Comment (xiao-mang) [reply bsp/nxp/imx/imx6ull-smart/applications/mnt.c:23]: 后面也是要支持emmc挂载文件系统的

Comment (BernardXiong) [reply bsp/nxp/imx/imx6ull-smart/applications/mnt.c:23]: 那就后面再加，代码都是有历史的，从历史版本中找回来即可。

Comment (xiao-mang) [reply bsp/nxp/imx/imx6ull-smart/applications/mnt.c]: 好的

File: components/dfs/dfs_v2/src/dfs_file_mmap.c (hunk 2 above)
Comment (xiao-mang) [reply components/dfs/dfs_v2/src/dfs_file_mmap.c]: 好的

**Review by Rbb666 — APPROVED** [review]

**Review by BernardXiong — APPROVED** [review]

**Review by geniusgogo — APPROVED** [review]

**Pull request merged**